cylc/flow/task_pool.py

```diff
@@ -422,8 +422,12 @@ def load_db_task_pool_for_restart(self, row_idx, row):
                     for message in json.loads(outputs_str).values():
                         itask.state.outputs.set_completion(message, True)
                 except (AttributeError, TypeError, ValueError):
-                    # Back compat for <=7.6.X
-                    # Each output in separate line as "trigger=message"
+                    # BACK COMPAT: outputs in separate lines
+                    # (as "trigger=message")
+                    # from:
+                    #     <=7.6.X
+                    # remove at:
+                    #     ???
```

Comment:
Thoughts???

Reply:
This is in `load_db_task_pool_for_restart()` and we're not support restart from a Cylc 7 DB, so can remove this, right?